### ISSUE TITLE
TableViewer navigation is not working if tree viewer is used in the columns

### ISSUE THREAD

**Issue by franzTobiasDLR** [virtualsatellite/VirtualSatellite4-Core#196]
The keyboard navigation for the table viewer in all AUiSnippetGenericTable does not work if a TreeViewer is used instead of a TableViewer. The table is not editable then.

Reproduce:
Eg. open a CEF mass parameter table
